packages/babel-helpers/src/helpers/regenerator.ts

```diff
@@ -328,10 +328,18 @@ export default function /* @no-mangle */ _regenerator() {
 
         if (_type > 3 /* Finish */) {
           if ((shouldReturn = finallyLoc === _arg)) {
-            method = entry[4]! || OperatorType.Jump;
-            arg = entry[5] === undefined ? entry[3]! : entry[5];
-            entry[4] = OperatorType.Jump;
-            entry[5] = undefined;
+            // if ((method = entry[4]!)) {
+            //   arg = entry[5];
+            // } else {
+            //   method = OperatorType.Jump;
+            //   arg = entry[3];
+            // }
```

Comment:
Do you think this is useful?

Reply:
Maybe add a note that the following line is a golfed version of this snippet? Otherwise at first glance they looks like some unused code ready for the cleanup.